include/openssl/evp.h

```diff
@@ -1166,6 +1166,11 @@ int EVP_MAC_CTX_get_params(EVP_MAC_CTX *ctx, OSSL_PARAM params[]);
 int EVP_MAC_CTX_set_params(EVP_MAC_CTX *ctx, const OSSL_PARAM params[]);
 
 size_t EVP_MAC_CTX_get_mac_size(EVP_MAC_CTX *ctx);
+unsigned char *EVP_Q_mac(OSSL_LIB_CTX *libctx, const char *name, const char *propq,
+                         const char *digest, const OSSL_PARAM *params,
```

Comment:
digest is definitely the wrong name here :)

Reply:
Oops, I'll make this (and as well the occurrences in `EVP_MAC.pod` consistent with the above.
That is currently is `subalg`, but maybe there is a better suggestion - see https://github.com/openssl/openssl/pull/14664#discussion_r613776594.

Reply:
Done (already a week ago).